app/com/arpnetworking/metrics/portal/scheduling/impl/MapJobRepository.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2018 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.portal.scheduling.impl;
+
+import com.arpnetworking.metrics.portal.scheduling.JobRepository;
+import com.arpnetworking.steno.Logger;
+import com.arpnetworking.steno.LoggerFactory;
+import com.google.common.collect.Maps;
+import com.google.inject.Inject;
+import models.internal.Organization;
+import models.internal.scheduling.Job;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * A simple in-memory {@link JobRepository}. Not in any way persistent, probably not good for production usage.
```

Comment:
Since `JobRespository` is intended as an interface only and not a physical implementation, this implementation is likely only for testing. Should we move it into the `src/test` directory?